app/views/shared/_savings_and_investments.html.erb

```diff
@@ -1,28 +1,16 @@
 <% if @cfe_result.liquid_capital_items.any? %>
-  <% if @legal_aid_application.passported? %>
-    <div class="govuk-accordion__section-content">
-      <h2 class="govuk-heading-m"><%= t('.savings_and_investments') %></h2>
-      <table class="govuk-table">
-        <tbody>
-        <% @cfe_result.liquid_capital_items.reject {|i| i[:description] == 'Online savings accounts' || i[:description] == 'Online current accounts' }&.map do |item| %>
-          <%= render partial: 'shared/results_detail_row', locals: { caption: item[:description], value: number_to_currency(item[:value]) } %>
+  <div class="govuk-accordion__section-content">
+    <h2 class="govuk-heading-m"><%= t('.savings_and_investments') %></h2>
+    <table class="govuk-table">
+      <tbody>
+        <% @cfe_result.liquid_capital_items.each do |item| %>
+      <% item %>
```

Comment:
This was a debug thing and can probably come out, even though it's not visible on page no `<%=`